examples/03_connectivity/plot_seed_to_voxel_correlation.py

```diff
@@ -133,48 +133,55 @@
 # series.
 import numpy as np
 
-seed_based_correlations = np.dot(brain_time_series.T, seed_time_series) / \
-                          seed_time_series.shape[0]
+seed_to_voxel_correlations = np.dot(brain_time_series.T, seed_time_series) / \
+                             seed_time_series.shape[0]
```

Comment:
For clarity and PEP8, rewrite this as:
(using parentheses instead of backslash for multi-line statements; & closing parenthesis on separate lines).
```
seed_to_voxel_correlations = (np.dot(brain_time_series.T, seed_time_series) 
                              / seed_time_series.shape[0]
                              ) 

```

Reply:
done